src/foam/u2/FoamTagLoader.js

```diff
@@ -57,4 +57,4 @@ foam.CLASS({
   ]
 });
 
-foam.u2.FoamTagLoader.create();
+//foam.u2.FoamTagLoader.create();
```

Comment:
Please exclude this from this pull request.

Also what do you need this change for?  Perhaps this file should receive it's own flag in files.js so we can avoid loading it in all contexts except a visible web page.